README.md

```diff
@@ -1 +1,77 @@
-# mobile-portfolio
\ No newline at end of file
+# Portfolio
+
+The project is inspired by the figma microverse template 1 (https://figma.com/)
+The goal of this project is to learn how parse a Figma design to create a UI, and use Flexbox to place elements in the page.
+
+
+
+- Screenshoot I
+
+![screenshot](./image/mob1.PNG)
+
+
+- Screenshoot II
+![screenshot](./image/mob2.PNG)
+
+
+- Screenshoot III
+![screenshot](./image/mob3.PNG)
+
+
+- set up a new repository and prepare it for development using best practices.
+- build mobile website.
+ 
+
+## Live Demo
+
+[Live Demo Link](https://mwanawabangona.github.io/mobile-portfolio/)
+ 
+## Built With
+
+- HTML
+- CSS
+- HTML&CSS Linter
+
+## Get Started
+
+Please get your browser update.
+To get a local copy up and running follow these simple steps.
+
+Run the command below from the Terminal:
+
+      git clone <URL> in your terminal
+
+	  cd microverse_portfolio
+
+	  start index.html
```

Comment:
- [ ] Be specific with the `<URL>` users have to clone to get started with your project.
- [ ] what do you mean by start `index.html`, kindly change this with open the live server while you are in the index.html file